src/lib/mongoBackend/mongoNotifyContextAvailability.cpp

```diff
@@ -46,10 +46,11 @@ HttpStatusCode mongoNotifyContextAvailability
 )
 {
     const std::string notifyFormat = uriParam[URI_PARAM_NOTIFY_FORMAT];
+    bool              reqSemTaken;
 
     LM_T(LmtMongo, ("Notify Context Availability: '%s' format", notifyFormat.c_str()));
 
-    reqSemTake(__FUNCTION__, "mongo ngsi9 notification");
+    reqSemTake(__FUNCTION__, "mongo ngsi9 notification", SemReadOp, &reqSemTaken);
```

Comment:
I think mongoNotifyContextAvailabiltity refers to the processing of notifyContextRequests. Thus, it is like a registerContext. In that case, SemWriteOp should be used.